### PR TITLE
Updated and added new data models.

### PR DESCRIPTION
**Pull request by yahoo-che**
Data models based on @swarupe04 schema

### CODE REVIEW

File: src/database/scrape-attempt.js [hunk 1]
```diff
@@ -1,8 +1,15 @@
 const { Schema } = require('mongoose');
 
 module.exports = new Schema({
```
Comment (swarupe04) [inline src/database/scrape-attempt.js:3]: this model we would need to update to include list of pdfs field with { link, hash }

**Review by swarupe04 — APPROVED** [review]

File: src/database/scrape-attempt.js (hunk 1 above)
Comment (yahoo-che) [reply src/database/scrape-attempt.js:3]: `urls: [new Schema({ url: String, hash: String })],`

Comment (swarupe04) [reply src/database/scrape-attempt.js:3]: hmm..i think we still want to separate out main link from pdf links. url: string, pdfs: [new Schema({url, hash})]

**Review by swarupe04 — APPROVED** [review]

**Review by ajit-gauli-vdms — COMMENTED** [review]
Would it make sense to have an active field in user schema that defaults to true, but can be set to false for users who want to opt-out of notifications?

**Comment by yahoo-che** [comment]
@ajit-gauli-vdms we got rid of the user notification model since there will only be a few users.

**Pull request merged**